src/test/integration/web_interface/rest/test_rest_binary.py

```diff
@@ -12,15 +14,15 @@
 class TestRestDownload(RestTestBase):
     def setup(self):
         super().setup()
-        self.db_interface = BackendDbInterface(self.config)
+        self.db_interface = BackendDbInterface()
         self.test_queue = Queue()
 
     def teardown(self):
         self.test_queue.close()
 
+    @pytest.mark.skip(reason='TODO')
```

Comment:
this hangs because the `try` is at the wrong position: the try block should start directly after creating the `InterComBackEndBinding` (otherwise the intercom is never shut down in case of an error)
In this case the error is that `self.tmp_dir` does not exist